indigo/analysis/work_detail/base.py

```diff
@@ -47,7 +47,7 @@ def work_friendly_type(self, work):
         return _('Act')
 
 
-class ChapterMixin:
+class ChapterWorkDetail(BaseWorkDetail):
```

Comment:
What's the motivation behind this?

Reply:
The ChapterMixin's `work_numbered_title` wasn't getting called at all. Chapters' numbered titles were being given as "Act x of yyyy"